src/eventmaster/api_handlers.go

```diff
@@ -121,6 +121,9 @@ func wrapHandler(h httprouter.Handle) httprouter.Handle {
 		start := time.Now()
 		defer func() {
 			httpReqLatencies.WithLabelValues(r.URL.Path).Observe(trackTime(start))
+			if rec := recover(); rec != nil {
+				fmt.Println("Panic from path:", r.URL.Path, rec)
```

Comment:
log? This should also probably write something out to the caller (as of now it'll just return empty--- maybe even a 200?)